EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001431	I am worried since the number of monsters increased a lot recently.{nl}They never came down this far...
+QUEST_LV_0300_20160310_001432	He didn't even bat an eye when they are injured.{nl}I hope the next lieutenant is nicer.
 QUEST_LV_0300_20160310_001433	The villagers are giving up on their work because the number of monsters increased a lot.{nl}I know it's an order, but it is so unfortunate.
-QUEST_LV_0300_20160310_001434	No matter how hard we have tried.{nl}I cannot accept that we have to leave this place..
-QUEST_LV_0300_20160310_001435	Thank  you!{nl}I can work again because of you!
-QUEST_LV_0300_20160310_001436	It was strange.{nl}But, we waited because we thought there must be some reason��{nl}
-QUEST_LV_0300_20160310_001437	We were being used, but we should regret the fact that we just waited for the order.{nl}I want to apologize to the villagers if I have opportunities.
-QUEST_LV_0300_20160310_001438	I can't do this anymore.{nl}I tried to start again here with everything I've got.. But, I don't know what happened.
+QUEST_LV_0300_20160310_001434	No matter how hard we tried.{nl}I cannot accept that we have to leave this place...
+QUEST_LV_0300_20160310_001435	Thank you!{nl}I can work again because of you!
+QUEST_LV_0300_20160310_001436	It was strange.{nl}We waited because we thought there must be some reason...{nl}
+QUEST_LV_0300_20160310_001437	We were being used, but we should regret the fact that we just waited for the order.{nl}I want to apologize to the villagers if I have the opportunity.
+QUEST_LV_0300_20160310_001438	I can't do this anymore.{nl}I tried to start again here with everything I've got... But, I don't know what happened.
 QUEST_LV_0300_20160310_001439	Thank you for helping me go back to the workplace.{nl}The humans are more scary than the monsters sometimes.
-QUEST_LV_0300_20160310_001440	The soldiers who've participated were already evacuated and it's the agent turn next.{nl}He thought about using the monsters.. It would've been better if he used it for good purposes.
-QUEST_LV_0300_20160310_001441	He received a big punishment from the landlord.{nl}He hid lots of corals at his bedroom.
-QUEST_LV_0300_20160310_001442	The research at Alremet Forest
-QUEST_LV_0300_20160310_001443	The request for the secretive rescue(1)
-QUEST_LV_0300_20160310_001444	Telling him I did according to the manual
-QUEST_LV_0300_20160310_001445	The request for the secretive rescue(2)
+QUEST_LV_0300_20160310_001440	The soldiers who've participated were already evacuated and it's the lieutenant's turn next.{nl}He thought about using the monsters... It would've been better if he used it for good purposes.
+QUEST_LV_0300_20160310_001441	He received a big punishment from the lord.{nl}He hid a lot of corals in his bedroom.
+QUEST_LV_0300_20160310_001442	The Research at Alemeth Forest
+QUEST_LV_0300_20160310_001443	Request for the Secret Rescue (1)
+QUEST_LV_0300_20160310_001444	I did according to the manual
```

Comment:
either: "I did, according to the manual" or "I did everything according to the manual"